### ISSUE TITLE
allow admins to do `layoutsave`

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2418]
This should save the current layout to the admin's machine, even if server is not in ` /devmap`. The reason why it should be reserved to admins is, this could otherwise be used to cheat.

What it would bring is, it would make it much easier to reproduce bugs seen only in specific layouts.

**Comment by DolceTriade** [comment]
Sure, seems reasonable

**Comment by sweet235** [comment]
If we do this, I'm quite sure that most servers will disable it. I certainly do not want people without ssh access to be able to create files. How would they fix a mistake?

**Comment by bmorel** [comment]
> I think we are talking about saving the layout on a client. That is fine with me.

Yes, obviously, as I would not want my server to be DoS by storage saturation :)
